tools/authors.sh

```diff
@@ -7,13 +7,11 @@ help() {
 	echo "  Generate an alphabetically ordered list of authors and update the authors file"
 	echo ""
 	echo "  Options:"
-	echo "    -c		Checks the authors file is valid without modifying it."
```

Comment:
@atenart, what about renaming this to -n (or any of your preference) and keeping it as a dry-run, instead?

Reply:
I don't think we should keep it as dry run, as it's not only checking the different steps can be run but is also checking the end result. If you think we should keep this, we can keep it as is IMO.

Reply:
> I don't think we should keep it as dry run, as it's not only checking the different steps can be run but is also checking the end result. If you think we should keep this, we can keep it as is IMO.

I mean something like this (adding  `-u` to `diff`):
```diff
❯ ./tools/authors.sh -c
--- /dev/fd/63  2024-04-19 16:08:45.856496295 +0200
+++ /dev/fd/62  2024-04-19 16:08:45.856496295 +0200
@@ -3,4 +3,8 @@
 Many thanks to everyone who contributed to Retis!
 
 In alphabetical order:
+- Adrian Moreno
+- Antoine Tenart
+- Daniel Mendes
 - Hangbin Liu
+- Paolo Valerio
```

Reply:
`diff -u` is quite nice, thanks for the suggestion!

> about dry run or check, I'm fine with both. Up to you.

On a second thought, you're 100% right, this is a dry run. Let's go with that.